examples/spring-boot-with-gradle-on-openshift-example/src/main/groovy/io/ap4k/example/sbonopenshift/Controller.groovy

```diff
@@ -0,0 +1,13 @@
+package io.ap4k.example.sbonopenshift
+
+import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.bind.annotation.RequestMapping
+
+@RestController
+class Controller {
```

Comment:
If we are shooting for the smallest possible example, maybe this should part of the main class?

It's obviously a super minor thing, so feel free to disregard it :P